commands/team_test.go

```diff
@@ -108,3 +108,250 @@ func (s *MmctlUnitTestSuite) TestCreateTeamCmd() {
 		s.Require().Len(printer.GetLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRemoveUserCmd() {
+	teamArg := "example-team-id"
+	userArg := "example-user-id"
+	s.Run("Remove users from team, without args returns an error", func() {
+		printer.Clean()
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{})
+		s.Require().Equal(err, errors.New("Not enough arguments."))
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Remove users from team, with one arg returns an error", func() {
+		printer.Clean()
+
+		err := removeUsersCmdF(s.client, &cobra.Command{}, []string{teamArg})
+		s.Require().Equal(err, errors.New("Not enough arguments."))
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Remove users from team, team not found, returns an error", func() {
```

Comment:
Let's change it to - "Remove users from team with a non-existent team returns an error".